iotlabaggregator/connections.py

```diff
@@ -77,7 +77,7 @@ def handle_close(self):
 
     def handle_read(self):
         """ Append read bytes to buffer and run data handler. """
-        self.data_buff += self.recv(8192)
+        self.data_buff += (self.recv(8192)).decode('utf-8-', 'replace')
```

Comment:
"utf-8-" au lieu de "utf-8" ?

Reply:
yes I will fix that and it's not utf-8 for sniffer